spec/services/collators/housing_costs_collator_spec.rb

```diff
@@ -11,47 +11,80 @@ module Collators
 
       subject(:collator) { described_class.call(assessment) }
 
-      context "no housing cost outgoings" do
-        it "records zero" do
-          collator
-          expect(disposable_income_summary.gross_housing_costs).to eq 0.0
-          expect(disposable_income_summary.housing_benefit).to eq 0.0
-          expect(disposable_income_summary.net_housing_costs).to eq 0.0
+      context "with no housing cost outgoings" do
+        context "without housing benefit" do
+          it "has expected housing cost attributes" do
+            collator
+            expect(disposable_income_summary)
+              .to have_attributes(
+                gross_housing_costs: 0.0,
+                housing_benefit: 0.0,
+                net_housing_costs: 0.0,
+              )
+          end
+        end
+
+        # TODO: missing spec? remove? - this tests existing functionality which shows it is possible
```

Comment:
question for others - BA has suggested we ensure net housing cost can never be less than zero.